bciers/libs/actions/src/api/postProductionData.ts

```diff
@@ -0,0 +1,15 @@
+import { actionHandler } from "../actions";
+import { ProductData } from "../../../types/src/form/productionData";
```

Comment:
"Consider updating the import statement to use the alias path for consistency and clarity:

import { Product, ProductData } from "@bciers/types/form/productionData";